Lookup gzdoom pk3 files in the installed directory in
addition to the path that is stored in ~/.config/gzdoom/.

Simply replacing the hardcoded /usr/share would leave the
configfile pointing to outdated guix builds, which could get
garbage collected and break the gzdoom installation.
This ensures that the files installed with the corresponding
build will be used.

--- /src/d_main.cpp	2018-01-16 22:13:09.700602781 +0200
+++ /src/d_main.cpp	2018-01-16 22:22:36.560617961 +0200
@@ -1716,6 +1716,11 @@
 		return wad;
 	}
 
+    mysnprintf (wad, countof(wad), "%s/%s", GUIX_OUT_PK3, file);
+    if (DirEntryExists (wad)) {
+        return wad;
+    }
+
 	if (GameConfig != NULL && GameConfig->SetSection ("FileSearch.Directories"))
 	{
 		const char *key;
